penny_university/settings/ci.py

```diff
@@ -1,7 +1,11 @@
+import os
+
 from penny_university.settings.base import *
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
 # If set to True, this ensures that any background tasks are run immediately rather than using django_background_tasks
 TASK_ALWAYS_EAGER = True
+
+FRONT_END_HOST = os.getenv('FRONT_END_HOST', 'http://localhost:3000')
```

Comment:
This is used in the `build_verification_url` method. I'm not sure if this is the best way to go about this, and would love some feedback.

Reply:
Looks good to me

Reply:
I need this on the backend too https://github.com/penny-university/penny_university/pull/226/files#r429599376 - what's a good generic name for holding this `https://www.pennyuniversity.org` and this `http://localhost:3000`?

Reply:
@JnBrymn eventually they should point to the same place. For now I think it would be fine to just create another env var and then we can clean it up once we are serving the SPA from a django route.